shared/modules/persistence_module.go

```diff
@@ -1,6 +1,6 @@
 package modules
 
-//go:generate mockgen -destination=./mocks/persistence_module_mock.go github.com/pokt-network/pocket/shared/modules PersistenceModule,PersistenceRWContext,PersistenceReadContext,PersistenceWriteContext,PersistenceLocalContext
+//go:generate mockgen -destination=./mocks/persistence_module_mock.go github.com/pokt-network/pocket/shared/modules PersistenceModule,PersistenceRWContext,PersistenceReadContext,PersistenceWriteContext,PersistenceLocalContext,TreeStoreModule
```

Comment:
any reason we are mocking it within persistence now it is a submodule?

Reply:
I believe I have some tests that aren't in PRs yet that will use mocked TreeStores.

Reply:
I meant moreso about the location of the mock `persistence_module_mock.go` vs `treestore_module_mock.go`

Reply:
Oh, I see what you meant now. Updated to put it in treestore_module instead.